### ISSUE TITLE
"unique symbol" is generated to multiple .d.ts files

### ISSUE THREAD

**Issue by zoltan-mihalyi** [microsoft/TypeScript#40786]
**TypeScript Version:**  4.0.3
**Search Terms:**
 unique symbol, declaration, .d.ts, invalid, TS4023

**Code**

*a.ts*
```ts
type AX = { readonly A: unique symbol };
export const A: AX = 0 as any;
```

*b.ts*
```ts
import { A } from './a';
export const A1 = A;
```
*tsconfig.json*
```json
{
  "compilerOptions": {
    "declaration": true,
    "strict": true,
    "noImplicitAny": true,
    "rootDir": "src",
    "outDir": "lib"
  }
}
```

**Expected behavior:**
Typescript should either throw TS4023, or generate the following code for b.d.ts:
 ```ts
import { A } from './a';
export declare const A1: typeof A;
```

**Actual behavior:**
Typescript generates "unique symbol" to both a.d.ts and b.d.ts:

*a.d.ts*
```ts
declare type AX = {
    readonly A: unique symbol;
};
export declare const A: AX;
export {};
```

*b.d.ts*
```ts
export declare const A1: {
    readonly A: unique symbol;
};
```

**Project Link:**
[project.zip](https://github.com/microsoft/TypeScript/files/5286468/project.zip)

**Comment by weswigham** [comment]
Hm. I suppose we should issue an error here? The local type from `a` isn't reachable in `b`, so we print the type anonymously; however it contains a `unique symbol` which, by rights, refers to a symbol in a different file; so we can't _really_ serialize it. The only thing you _could_ write out is something like `export declare const A1: typeof A`, like you propose, but our declaration emit _generally_ doesn't write out `typeof` queries on its own (simply because it doesn't generalize - it'd only work for exactly declarations of the form `var x = some.dotted.name`, which is an odd syntactic carve-out to make for TS code), which is why it's more in-line with our normal behavior to report an error on the attempted usage of a private symbol, and require an annotation.

**Comment by weswigham** [comment]
In fact, the error already exists, and is simply failing to trigger due to an unfortunate bug.
